deploy/roles/r-core/defaults/main.yml

```diff
@@ -0,0 +1,8 @@
+---
+r_core_state: ""
+
+- include_role: 
+    name: "{{ item }}"
+  with_items:
+    - r-core
+    - r-libs
```

Comment:
@lel99999 Sorry, I didn't mean here, `defaults/main.yml` is just for default variables.  `r-core` and `r-libs` should be listed in the playbooks themselves, outside of the roles, like in https://github.com/cfpb/aurora/blob/develop/deploy/deploy_r_terminal.yml where it current has just `- r`.